.pre-commit-config.yaml

```diff
@@ -64,6 +64,7 @@ repos:
           - "types-lxml"
 
         pass_filenames: false
+        args: [--config-file=pyproject.toml]
```

Comment:
This revert shouldn't be necessary, it's not related to the warning. It's just there, in the same PR, a dropped line that is not needed, and illustrates why adding unrelated changes in a PR may make things more complicated for the future readers.